src/tests/class_tests/openms/source/SpectrumExtractor_test.cpp

```diff
@@ -443,6 +437,147 @@ START_SECTION(pickSpectrum())
 }
 END_SECTION
 
+START_SECTION(annotateSpectrum())
+{
+  MzMLFile mzml;
+  PeakMap experiment;
+  TargetedExperiment targeted_exp;
+  TransitionTSVReader tsv_reader;
+
+  // targeting a particular mz and rt
+  //double target_mz = 177.0;
+  //double target_rt = 840.0;
+
+  ptr->setRTWindow(30);
+  ptr->setMZTolerance(0.1);
+  ptr->setGaussWidth(0.25);
+  ptr->setUseGauss(true);
+
+  // mzml.getOptions().setRTRange(DRange<1>(target_rt - ptr->getRTWindow() / 2.0, target_rt + ptr->getRTWindow() / 2.0));
+  // mzml.getOptions().setRTRange(DRange<1>(0.0, 856));
+  // mzml.getOptions().setMZRange(DRange<1>(target_mz - ptr->getMZTolerance(), target_mz + ptr->getMZTolerance()));
+  // mzml.getOptions().setMZRange(DRange<1>(0.0, 300.0));
+
+  // TODO remove the following path, use different way to load data
+  mzml.load("/home/pasdom/Downloads/douglas_files/13C1.mzML", experiment);
```

Comment:
Try using this `OPENMS_GET_TEST_DATA_PATH("AbsoluteQuantitationMethodFile_1.csv")` instead of the absolute file path.  However, it may not be good to include such a large file as an actually test.  Perhaps we should think about removing some of the specrum to generate a shorter file for the test.